packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1106,6 +1135,20 @@ export class ContainerRuntime
 		this.maxOpsSinceLastSummary = this.getMaxOpsSinceLastSummary();
 		this.initialSummarizerDelayMs = this.getInitialSummarizerDelayMs();
 
+		// Enabling the compressor is a one-way operation. Once it's enabled for a document
+		// it is persisted in the container's metadata.
+		const compressorEnabled =
```

Comment:
Can you add `compressorEnabled` to `ContainerLoadStats` telemetry below. That way, we will know if it is enabled for a container or not.